test/test.js

```diff
@@ -2901,6 +2901,12 @@ describe('Chat', () => {
 				expect(data.file).to.be.not.empty;
 			});
 
+			it('Upload a buffer', async () => {
```

Comment:
why didn't you add the other test in stream-js?

https://github.com/GetStream/stream-js/pull/389/files#diff-b3be5f00e4539d897e017e0426fb7606R22

Reply:
pushed a similar test 👍